tests/common.rs

```diff
@@ -7,3 +7,9 @@ pub fn eval(s: impl std::string::ToString) -> Result<Term, Error> {
     let mut p = Program::new_from_source(Cursor::new(s.to_string()), "test").unwrap();
     p.eval()
 }
+
+pub fn eval_file(f: &str) -> Result<Term, Error> {
+    let mut p =
+        Program::new_from_file(format!("{}/tests/{}", env!("CARGO_MANIFEST_DIR"), f)).unwrap();
```

Comment:
Although it is probably not important for tests, why `unwrap()` when we can bubble up the error using `?`

Reply:
Because we can not right now ;) It's the same reason we use `unwrap` in the other function of this module. `Program::new_from_...` functions return a `std::io::Result`. It will be better to replace it by a `Result<_, errors::IOError>` but does not concern this PR I think and is not a small change. I'll open an issue for that.

Reply:
I thought there was a top-leve `IOError`, but it's only for imports. That makes sense